docs/items/weapon-configs.md

```diff
@@ -487,16 +487,16 @@ The configurations that make up each of the custom weapons.
 ## [38] Blaze Hydra
```

Comment:
Remove the blaze hydra

Reply:
Rremoved, and replaced Blaze Hydra slot with the Fuel Rod